test_autolens/lens/test_dict.py

```diff
@@ -21,7 +21,7 @@ def make_tracer():
     source_galaxy = al.Galaxy(redshift=1.0, disk=disk)
 
     return al.Tracer.from_galaxies(
-        galaxies=[lens_galaxy, source_galaxy], cosmology=al.cosmo.Planck15()
+        galaxies=[lens_galaxy, source_galaxy], cosmology=al.cosmo.wrap.Planck15
```

Comment:
This actually gives a lint error because al.cosmo.wrap.Planck15 is a type not an instance of that type